src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -148,15 +148,17 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
       <ProjectEditDetails users={users} control={control} errors={errors} />
       <PageBlock title="Project Summary">
         <Grid item sx={{ mt: 2 }}>
-          <ReactHookTextField
-            name="summary"
-            control={control}
-            sx={{ width: '50%' }}
-            label="Summary"
-            multiline={true}
-            rows={5}
-            errorMessage={errors.summary}
-          />
+          <Typography>
```

Comment:
same idea as external link component. if the ReactHookTextField component definitely needs usage of typography, we would do that inside the component, not wrapping it